[2025-07-27T10:13:10.079+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-27T10:13:10.359+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_setup_teardown_taskflow.section_1.inner_work scheduled__2025-07-26T00:00:00+00:00 [queued]>
[2025-07-27T10:13:10.517+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_setup_teardown_taskflow.section_1.inner_work scheduled__2025-07-26T00:00:00+00:00 [queued]>
[2025-07-27T10:13:10.527+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-27T10:13:10.622+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): section_1.inner_work> on 2025-07-26 00:00:00+00:00
[2025-07-27T10:13:10.721+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=326) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-27T10:13:10.744+0000] {standard_task_runner.py:64} INFO - Started process 433 to run task
[2025-07-27T10:13:10.761+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_setup_teardown_taskflow', 'section_1.inner_work', 'scheduled__2025-07-26T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_setup_teardown_taskflow.py', '--cfg-path', '/tmp/tmp3fidwnxc']
[2025-07-27T10:13:10.789+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask section_1.inner_work
[2025-07-27T10:13:10.913+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-27T10:13:11.249+0000] {task_command.py:426} INFO - Running <TaskInstance: example_setup_teardown_taskflow.section_1.inner_work scheduled__2025-07-26T00:00:00+00:00 [running]> on host 4112a2618ff3
[2025-07-27T10:13:12.300+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_setup_teardown_taskflow' AIRFLOW_CTX_TASK_ID='section_1.inner_work' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-26T00:00:00+00:00'
[2025-07-27T10:13:12.318+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-27T10:13:12.341+0000] {logging_mixin.py:188} INFO - doing some work with cluster_id='some_cluster_id'
[2025-07-27T10:13:12.352+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-27T10:13:12.356+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-27T10:13:12.387+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_setup_teardown_taskflow, task_id=section_1.inner_work, run_id=scheduled__2025-07-26T00:00:00+00:00, execution_date=20250726T000000, start_date=20250727T101310, end_date=20250727T101312
[2025-07-27T10:13:12.615+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-07-27T10:13:12.922+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-27T10:13:12.932+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
